src/PostgREST/ApiRequest.hs

```diff
@@ -122,6 +122,10 @@ userApiRequest schema req reqBody =
               Nothing -> PayloadParseError "All lines must have same number of fields"
               Just json -> PayloadJSON json)
             (CSV.decodeByName reqBody)
+        Left "application/x-www-form-urlencoded" ->
```

Comment:
got it.
In this case i think it is ok as it is now, maybe just a comment for this special case and why the choice was made.
It's not worth putting in more work/lines of code to support this feature and it's only a minimal change.